components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -1,13 +1,121 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import dayjs from 'dayjs';
+import dynamic from 'next/dynamic';
+import { FormattedMessage, useIntl } from 'react-intl';
 
 import { formatCurrency } from '../../../lib/currency-utils';
 
-import { Box } from '../../Grid';
+import Container from '../../Container';
+import { Box, Flex } from '../../Grid';
 import LoadingPlaceholder from '../../LoadingPlaceholder';
 import ProportionalAreaChart from '../../ProportionalAreaChart';
 import { P, Span } from '../../Text';
+const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
+
+const getChartOptions = (intl, startDate, endDate, hostCreatedAt) => {
+  return {
+    chart: {
+      id: 'chart-transactions-overview',
+    },
+    legend: {
+      show: true,
+      horizontalAlign: 'left',
+    },
+    colors: ['#29CC75', '#F55882'],
+    grid: {
+      xaxis: { lines: { show: true } },
+      yaxis: { lines: { show: false } },
+    },
+    stroke: {
+      curve: 'straight',
+      width: 1.5,
+    },
+    dataLabels: {
+      enabled: false,
+    },
+
+    xaxis: {
+      categories: getCategories(intl, startDate, endDate, hostCreatedAt),
+    },
+    yaxis: {
+      labels: {
+        formatter: function (value) {
+          return value < 1000 ? value : `${Math.round(value / 1000)}k`;
+        },
+      },
+    },
+  };
+};
+
+const getCategories = (intl, startDate, endDate, hostCreatedAt) => {
+  const numberOfDays = dayjs(startDate || hostCreatedAt).diff(dayjs(endDate), 'day');
+  if (numberOfDays <= 7) {
+    const startDay = startDate.getDay();
+    return [...new Array(7)].map(
+      (_, idx) => `${intl.formatDate(new Date(0, 0, idx + startDay), { weekday: 'long' }).toUpperCase()}`,
```

Comment:
Why use a template string? Same for the other two below

```suggestion
      (_, idx) => intl.formatDate(new Date(0, 0, idx + startDay), { weekday: 'long' }).toUpperCase(),
```

Reply:
Ah, seems a mistake. 🤯 I've removed these.